ViAn/Analysis/analysismethod.cpp

```diff
@@ -131,7 +131,6 @@ void AnalysisMethod::run() {
     if(*aborted){
         capture.release();
         emit analysis_aborted();
-        emit finito();
```

Comment:
Was this signal not connected before? Can't find the removal of the connection line